EIPS/eip-2387.md

```diff
@@ -0,0 +1,61 @@
+---
+eip: 2387
+title: "Hardfork Meta: Mountain Glacier"
```

Comment:
Suggest "Muir Glacier" ([see here](https://en.wikipedia.org/wiki/Muir_Glacier)) which has retreated significantly. Or any other similar glacier :wink:

Reply:
Or perhaps a characters name from the movie Ice Age? 😂